components/nimbus/src/enrollment.rs

```diff
@@ -527,6 +527,16 @@ impl EnrollmentStatus {
     }
 }
 
+pub(crate) trait SharedExperimentMetadata {
+    fn get_slug(&self) -> String;
+
+    fn is_rollout(&self) -> bool;
+
+    fn is_not_rollout(&self) -> bool {
+        !self.is_rollout()
+    }
+}
```

Comment:
The `not` method is there because of how it is being used — @jhugman commented on #5475 that I can instead make a temporary variable that's a lambda for the same functionality so that's my plan